package.json

```diff
@@ -18,6 +18,7 @@
     "precompile": "npm run clean-lib && npm run copy-files && npm run babel",
     "prepublishOnly": "node check-version.js && npm install && npm run precompile",
     "watch": "npm run clean-lib && npm run copy-files -- --watch & npm run babel -- --watch",
+    "storybook": "start-storybook -p 9001 -c .storybook",
```

Comment:
is `start-storybook` a CLI that is required? or can we use an exe in node_modules?

Reply:
Good point, I've changed it. Thanks